library/src/plan.cpp

```diff
@@ -192,6 +265,12 @@ rocfft_status rocfft_plan_create_internal(rocfft_plan                   plan,
             {
                 if(description->inArrayType != description->outArrayType)
                     return rocfft_status_invalid_array_type;
+                if(description->inStrides != description->outStrides)
```

Comment:
double check this, I thought we removed it.

Reply:
The type has been changed from size_t* to std::array<size_t, 3>, so the check is on the data, not the address.  However, this isn't the correct place to check this, so we're probably just looking at the default values, so I removed it in d91b984.